### PR TITLE
Neo4j blue

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: organisations-rw-neo4j-blue@.service [hunk 1]
```diff
@@ -17,11 +17,10 @@ ExecStart=/bin/sh -c '\
   export APP_PORT=8080; \
   export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
   export NEO_HOST_PORT=$(/usr/bin/etcdctl get /ft/services/neo4j/servers/1); \
```
Comment (tosan88) [inline organisations-rw-neo4j-blue@.service]: This isn't needed anymore, is it?

Comment (catalin-ilea) [inline organisations-rw-neo4j-blue@.service]: Removed.

**Comment by tosan88** [comment]
:+1:

**Comment by Zapadlo** [comment]
+1

**Pull request merged**